cmd/flakechecker/flake_config.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/pkg/errors"
+	"io"
+	"os"
+	"regexp"
+)
+
+// flakeCheckerRecord represents configuration record used by flakechecker to evaluate failed tests
```

Comment:
```suggestion
// flakeCheckerRecordConfig represents configuration record used by flakechecker to evaluate failed tests.
```

Though I'm not sure it's the best name...